x-pack/legacy/plugins/index_management/public/app/components/template_form/template_form_schemas.tsx

```diff
@@ -68,14 +68,6 @@ export const schemas: Record<string, FormSchema> = {
             ),
           }),
         },
-        {
```

Comment:
Looks like your changes from https://github.com/elastic/kibana/pull/55145 ended up in this PR. Can you remove this code change?